pkg/crc/preflight/preflight_checks_common.go

```diff
@@ -43,6 +45,21 @@ var genericPreflightChecks = [...]PreflightCheck{
 		fix:              fixBundleCached,
 		flags:            SetupOnly,
 	},
+	{
+		configKeySuffix:  "check-ram",
+		checkDescription: "Checking minimum RAM requirements",
+		check:            checkRAM,
+		fixDescription:   "crc requires at least 9GB to run",
+		flags:            NoFix,
+	},
+}
+
+func checkRAM() error {
+	totalMemory := memory.TotalMemory()
+	if totalMemory < 9_000_000_000 {
```

Comment:
Can this check compare totalMemory against `crcConfig.GetInt(config.Memory.Name)` because it is possible to increase the memory setting?

https://github.com/code-ready/crc/blob/a3bd657a7c04292f59454e7dbfb62f54ac367b6d/cmd/crc/cmd/start.go#L55

https://github.com/code-ready/crc/blob/master/docs/source/topics/proc_configuring-the-virtual-machine.adoc

Reply:
Do you mean something like `totalMemory < max(9GB, crcConfig.GetInt(config.Memory.Name))` ? I think we really want to prevent user from running crc on too little machine (docs says `The amount of memory to assign must be greater than or equal to the default.`).

Reply:
Yes. It was just an idea to ensure if someone sets 16 GB we check totalMemory against this configuration.

You are right, setting a configuration with less than 9 GB (default) is already not possible:

```
➜  ~ crc config set memory 1024
ERRO requires memory in MiB >= 9216               
Value '1024' for configuration property 'memory' is invalid, reason: requires memory in MiB >= 9216
```

Reply:
If you have TOO little memory, Hyper-V will be prevented to start. And this NOT related to total memory, but free memory.